### ISSUE TITLE
[Ops] support for log

### ISSUE THREAD

**Issue by pilkicTT** [tenstorrent/tt-mlir#1098]
Blocks e2e training on device; needed for running `CrossEntropyLoss` loss function on device.

https://docs.tenstorrent.com/ttnn/latest/ttnn/ttnn/log.html
